vignettes/ORFikOverview.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "ORFik Overview"
+author: "Håkon Tjeldnes"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{ORFik Overview}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Welcome to the `ORFik` package. This vignette will walk you through our main 
+package usage with two examples, one easy and one real world example.  You will learn how to interpret 
+results, make features, and plot results.
+This package, `ORFik`, is created for fast and analysis of Open Reading frames, and to make less people reinvent the wheel.
+
+# Introduction
+
+`ORFik` main usage is to get Open reading frames from a fasta file and a gtf file describing the sequences, what you do with them, make features, plots etc, is up to you.
```

Comment:
again this is not main usage, its just one of the functions. Below this is how I imagine findORFs should be working:
To find all Open Reading Frames (ORFs) use `?findORFs` function. If input is only .fasta file (or BSGenome), `findORFs` will treat it as prokaryotic genome and findORFs on the chromosomes assuming no splicing. If .gtf file is also supplied, sequences of transcripts from .gtf file will be extracted and `findORFs` will search there. It is also possible to operate on a vector of characters or Biostrings::DNAString or Biostrings::DNAStringSet.